<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tên</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_ipt"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Giới TÍnh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gioitinh_ipt">
        <item>
         <property name="text">
          <string>Nam</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nữ</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Tuổi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="age_ipt"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Môn Dạy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mon_ipt">
        <item>
         <property name="text">
          <string>Văn</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Toán</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Anh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KHTN</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>GVCN Lớp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gvcn_ipt">
        <item>
         <property name="text">
          <string>Không có lớp trống</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
